plugins/destination/sqlite/client/transformer.go

```diff
@@ -27,11 +32,17 @@ func (*Client) TransformJSON(v *schema.JSON) interface{} {
 }
 
 func (*Client) TransformText(v *schema.Text) interface{} {
-	return v.String()
+	return stripNulls(v.String())
 }
 
 func (*Client) TransformTextArray(v *schema.TextArray) interface{} {
-	return v.String()
+	strippedStrings := make([]string, len(v.Elements))
```

Comment:
Nit: Given this function is heavily called, use string builder here instead of making two allocations. https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go and then calling `fmt.Sprintf`